README.md

```diff
@@ -1,20 +1,42 @@
 # 🎭 playwright-test
 
-> This package is not ready for use. If you want to run tests with playwright, checkout [jest-playwright](https://github.com/mmarkelov/jest-playwright) for Jest or [karma-playwright-launcher](https://github.com/JoelEinbinder/karma-playwright-launcher) for Karma.
+> 🚧 This project is under development. See [Test runner integrations](https://playwright.dev/#version=v1.4.1&path=docs%2Ftest-runners.md&q=) to use Jest or Mocha with Playwright.
```

Comment:
lets maybe use the non-version bound link?

Reply:
sure 👍